packages/cgb-scripts/package.json

```diff
@@ -30,10 +30,21 @@
 		"cgb-scripts": "./bin/cgb-scripts.js"
 	},
 	"dependencies": {
+		"@babel/core": "^7.4.4",
+		"@babel/plugin-proposal-class-properties": "^7.0.0",
+		"@babel/plugin-proposal-decorators": "^7.0.0",
+		"@babel/plugin-proposal-export-namespace-from": "^7.0.0",
+		"@babel/plugin-proposal-function-sent": "^7.0.0",
+		"@babel/plugin-proposal-json-strings": "^7.0.0",
+		"@babel/plugin-proposal-numeric-separator": "^7.0.0",
+		"@babel/plugin-proposal-throw-expressions": "^7.0.0",
+		"@babel/plugin-syntax-dynamic-import": "^7.0.0",
+		"@babel/plugin-syntax-import-meta": "^7.0.0",
+		"@babel/preset-env": "^7.4.4",
+		"@babel/preset-react": "^7.0.0",
 		"autoprefixer": "^7.2.4",
-		"babel-core": "^6.25.0",
-		"babel-eslint": "^8.2.1",
-		"babel-loader": "^7.1.1",
+		"babel-eslint": "^9.0.0",
+		"babel-loader": "^8.0.0",
 		"babel-preset-cgb": "^1.6.0",
```

Comment:
I believe we might need to update `babel-preset-cgb` as well.

Reply:
Check this https://github.com/ahmadawais/create-guten-block/blob/master/packages/babel-preset-cgb/index.js

Reply:
^ yes that's exactly what I was looking for that I was missing. I understand now how the preset is used for the build task before ejection, and then not used at all afterwards. It seems like it might be a better solution to grab the babel-preset-cgb and use that inside eject.js instead of duping this code in both places. 

Working on this change now.

Reply:
Go right ahead!